dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/loop/template/http/method/HttpLoopTaskCancelTaskMethodDefinition.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.plugin.task.api.loop.template.http.method;
+
+import org.apache.dolphinscheduler.plugin.task.api.loop.LoopTaskCancelMethodDefinition;
+import org.apache.dolphinscheduler.plugin.task.api.loop.LoopTaskInstanceInfo;
+import org.apache.dolphinscheduler.plugin.task.api.loop.template.http.HttpLoopTaskMethodDefinition;
+import org.apache.dolphinscheduler.plugin.task.api.utils.OkHttpUtils;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Map;
+
+public class HttpLoopTaskCancelTaskMethodDefinition extends HttpLoopTaskMethodDefinition
+    implements LoopTaskCancelMethodDefinition {
+
+    private final String taskInstanceIdHolder = "${taskInstanceId}";
+
+    public HttpLoopTaskCancelTaskMethodDefinition(String url,
+                                                  String httpMethodType,
+                                                  String dataType,
+                                                  Map<String, Object> requestParams,
+                                                  Map<String, Object> requestBody) {
+        super(url, httpMethodType, dataType, requestParams, requestBody);
+    }
+
+    @Override
+    public void cancelTaskInstance(LoopTaskInstanceInfo loopTaskInstanceInfo) {
+        if (requestParams != null) {
```

Comment:
It seems some same code.

Reply:
Yes, right now there are some same code in the queryStatus/cancelTask, but these two methods will be different.